packages/legacy/core/App/screens/Connection.tsx

```diff
@@ -246,6 +247,11 @@ const Connection: React.FC<ConnectionProps> = ({ navigation, route }) => {
       })
       return
     }
+
+    if ((state.notificationRecord as OpenId4VPRequestRecord).type === 'OpenId4VPRequestRecord') {
+      dispatch({ inProgress: false })
+      navigation.replace(Screens.OpenIDProofPresentation, { credential: state.notificationRecord })
```

Comment:
In future, can we avoid passing objects as navigation params? I think react-navigation recommends only using primitives as params, so we could pass a credential ID instead.

Reply:
we are working in a model that will create a react-hooks for open-id work, this then will be solved